modules/new_user/new_user.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  8 12:49:48 2021
+
+@author: pinny
+"""
+
+from discord import utils
+
+def is_new(member):
+    if utils.get(member.roles, name="Unassigned") != None:
+        return True
+    return False
+
+async def add_role(ctx, machon, year):
+        member = ctx.author
+    
+        #adjusts the name to be specific to a role and assigns the role object to new_role
+        role_name = machon.capitalize() + ' ' + year
+        new_role = utils.get(member.guild.roles, name = role_name)
+
+        #check if the role was legal
+        if new_role == None:
+            await ctx.send("Illegal Machon or Year. Please try again.")
+        
+        await member.add_roles(new_role)
+        print(f"Gave {new_role} to {member}")
+
+async def remove_unassigned(member):
+    role = utils.get(member.guild.roles, name = "Unassigned")
```

Comment:
Better to use IDs instead of names in case the names get changed.

With developer mode turned on, you can right click to get the role ID